s3/pkg/datastore/yig/storage/gc.go

```diff
@@ -70,6 +79,22 @@ func (gm *GcMgr) Start() {
 				}
 				log.Errorf("failed to remove object: %s, err: %s", result.ObjectId, result.Err)
 			}
+			gcEnd := time.Now()
```

Comment:
Hi @hopkings2008 Could you please add comment to explain the logic here? For example, what count and  backoffCound stand for, why need intervalTime and maxIntervalTime?

Reply:
@sfzeng thanks and i have added more comments for this logic, pls check it again.